app/src/main/res/values/strings.xml

```diff
@@ -20,6 +20,9 @@
 
     <string name="google_search">Searching the query from google</string>
     <string name="set_alarm">Setting the Alarm</string>
+    <string name="reminderDes">Write the description for the Reminder</string>
```

Comment:
As mentioned my @Abhi2424shek, follow the naming scheme followed throughout the app.